Backend/src/modules/users/users.service.ts

```diff
@@ -9,6 +9,21 @@ interface CreateUserInput {
   profilePicture: string | undefined;
   phone: string | undefined;
 }
+interface UpdateUserInfo {
+  id: string;
+  data: {
```

Comment:
why you didn't make data take type UserUpdateDTO (i don't remember the exact name). But I mean why you didn't use the dto type here?